docs/docsite/rst/user_guide/intro_adhoc.rst

```diff
@@ -114,6 +114,8 @@ Managing packages
 
 You might also use an ad hoc task to install, update, or remove packages on managed nodes using a package management module like yum. To ensure a package is installed without updating it:
 
+Package management modules support common functions to install, remove, and generally manage packages. Some specific functions for a package manager might not be present in the Ansible module since they are not part of general package management.
```

Comment:
```suggestion
Package management modules support common functions to install, remove, update, and other package management tasks. Some specific operations for a package manager might not be present in the Ansible module since these operations may not be part of general package management.
```

I would move on line 115, since line 119 code block is more relevant to 
```
 To ensure a package is installed without updating it:
```